### ISSUE TITLE
Failing to compile glutin

### ISSUE THREAD

**Issue by johnpmayer** [ryanisaacg/quicksilver#555]
This happens for me on darwin stable (1.40). But it's also failing on GitHub CI since the 0.4 commit 78572c066ff894fe8a8cf31f1e3f44042897e1b9.

```
   Compiling glutin v0.22.0-alpha5
error[E0107]: wrong number of type arguments: expected 1, found 0
   --> /Users/johnmayer1/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha5/src/context.rs:178:15
    |
178 |         size: dpi::PhysicalSize,
    |               ^^^^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> /Users/johnmayer1/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha5/src/platform_impl/macos/mod.rs:171:16
    |
171 |         _size: dpi::PhysicalSize,
    |                ^^^^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> /Users/johnmayer1/.cargo/registry/src/github.com-1ecc6299db9ec823/glutin-0.22.0-alpha5/src/windowed.rs:180:32
    |
180 |     pub fn resize(&self, size: dpi::PhysicalSize) {
    |                                ^^^^^^^^^^^^^^^^^ expected 1 type argument

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0107`.
error: could not compile `glutin`.

To learn more, run the command again with --verbose.
```

**Comment by johnpmayer** [comment]
Looks like you're already aware of it - https://github.com/grovesNL/glow/pull/69

**Comment by johnpmayer** [comment]
I think this is fixed with the release of 0.22.0

**Comment by ryanisaacg** [comment]
This should be fixed with blinds `v0.1.0-alpha9`. Feel free to re-open if it's not.
